rrmngmnt/host.py

```diff
@@ -335,14 +353,59 @@ def is_connective(self, tcp_timeout=20.0):
         :return: True if host is connective, False otherwise
         :rtype: bool
         """
-        ret = False
         try:
-            self.executor().run_cmd(['true'], tcp_timeout=tcp_timeout)
-            ret = True
-        except (socket.error, socket.timeout):
-            ret = False
-        except Exception:
-            self.logger.warning("Unexpected exception", exc_info=True)
-            ret = False
-
-        return ret
+            self.logger.info(
+                "Check if host %s is connective via ssh", self.fqdn
+            )
+            self.run_command(['true'], tcp_timeout=tcp_timeout)
+            return True
+        except (socket.timeout, socket.error) as e:
+            self.logger.debug("Socket error: %s", e)
+        except paramiko.SSHException as e:
+            self.logger.debug("SSH exception: %s", e)
+        self.logger.warning("Host %s does not connective via ssh", self.fqdn)
+        return False
+
+    def wait_for_ssh_connective_state(
+        self, positive, timeout=REBOOT_TIMEOUT, sample_time=REBOOT_SAMPLE_TIME
```

Comment:
SSH_TIMEOUT and REBOOT_SAMPLE_TIME. this function is about ssh connectivity to host and not about reboot